src/components/dependencies/Component.js

```diff
@@ -239,6 +241,39 @@ const Component = React.memo(({
     complianceSummaryState.error,
   ])
 
+  const [mountRescoring, setMountRescoring] = React.useState(searchParamContext.get('rescoreArtefacts'))
+
+  const artefactNodes = React.useMemo(() => {
+    if (!isParentComponent) return [] // only allow rescore linking for root component
+    const artefactIds = searchParamContext.getAll('rescoreArtefacts')
+    if (artefactIds.length === 0) return []
+
+    return component.resources.filter((resource) => {
+      if (artefactIds.includes(`${resource.name}:${resource.version}`)) return true
+      return false
+
+    }).map((resource) => {
+      return new OcmNode([component], resource, ARTEFACT_KIND.RESOURCE)
+
+    })
+  }, [
+    searchParamContext,
+    component.resources
+  ])
+
+  const handleRescoringClose = React.useCallback(() => {
+    setMountRescoring(false)
+    searchParamContext.delete('rescoreArtefacts')
+  }, [setMountRescoring])
+
+  const scanConfigName = searchParamContext.get('scanConfigName')
+  // eslint-disable-next-line no-unused-vars
+  const [scanConfigs, scanCfgState] = useFetchScanConfigurations()
```

Comment:
Why do you destruct two from three values here, although only `scanConfigs` is used. I would either expect all three being destructed (and hence the disabling of the linter), or only the one that is actually being required.